src/components/FlawAffects/__tests__/FlawAffects.spec.ts

```diff
@@ -233,6 +233,30 @@ describe('flawAffects', () => {
     expect(resolutionOptions.includes('DEFER')).toBe(false);
   });
 
+  osimFullFlawTest('Changing affect to not affected automatically set\'s empty resolution', async () => {
+    const affectsTableRows = subject.findAll('.affects-management table tbody tr');
+
+    let resolutionSelect = affectsTableRows[0].find('td:nth-of-type(6) span');
+    expect(resolutionSelect.text()).not.toEqual('');
+
+    const affectRowEditBtn = affectsTableRows[0].find('td:last-of-type button:first-of-type');
+    expect(affectRowEditBtn.exists()).toBe(true);
+    await affectRowEditBtn.trigger('click');
+
+    const affectsTableEditingRows = subject.findAll('.affects-management table tbody tr.editing');
+    const affectednessSelect = affectsTableEditingRows[0].find('td:nth-of-type(5) select');
+    await affectednessSelect.setValue('NOTAFFECTED');
+    const selectedAffectedness = affectednessSelect.find('option[selected]');
+    expect(selectedAffectedness.text()).toBe('NOTAFFECTED');
+
+    const affectRowCommitBtn = affectsTableEditingRows[0].find('td:last-of-type button:first-of-type');
+    expect(affectRowCommitBtn.exists()).toBe(true);
+    await affectRowCommitBtn.trigger('click');
+
+    resolutionSelect = affectsTableRows[0].find('td:nth-of-type(6) span');
```

Comment:
I think there's no need to reassign the element again, as this point to the DOM element that should be updated

Reply:
I think it is necessary, even if the DOM is updated the variable declared is a constant.
Removing last assignment makes the expect after it to fail.

Reply:
The variable being a constant only means it cannot be reassigned, but if the value is a reference it can update on its own, but you're right, the VueWrapper does not update on its own, you have to query it manually https://github.com/vuejs/test-utils/issues/1106